google/cloud/pubsub/internal/tracing_message_batch.cc

```diff
@@ -97,21 +97,21 @@ auto MakeChild(
 }
 
 Spans MakeBatchSinkSpans(Spans message_spans) {
-  int constexpr kMaxOtelLinks = 128;
+  int64_t max_otel_links = 128;
```

Comment:
`std::int64_t`.  (Aside: It looks like we should also `#include <cstdint>`.)

Reply:
Missed these comments before it went in. I will revisit them when I convert this to an option value. https://github.com/googleapis/google-cloud-cpp/issues/12979